### DISCUSSION TITLE
Wio LTE Cat.1 | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#204] [Q&A]
# Wio LTE Cat.1 | Seeed Studio Wiki

Wio Tracker (Wireless Input Output) is an open source gateway which enable faster IoT GPS solutions. It is Arduino and Grove compatible development boards that helps you track nearly any moving thing on the planet and then upload that data wirelessly. The Wio LTE is the LTE version of Wio Tracker, so now we’ve got 2 versions of Wio Tracker and the LTE (4G) version will make some differences.

https://wiki.seeedstudio.com/Wio_LTE_Cat.1

**Comment by matsujirushi** [comment]
Hi,

> Step 2. Follow How to Add Seeed boards to Arduino IDE to add Wio_LTE into arduino board manager.

Wio LTE board package not found.
Could you write the complete name of the board package on the wiki?

**Reply by MatthewJeffson** [reply]
Hi, the board of "Wio LTE Cat.1" is stored in "Seeed SAMD Boards" package like "Wio Terminal". I will update it soon.

**Reply by matsujirushi** [reply]
Really?
Wio LTE Cat.1 MCU is STM32 instead of SAMD.

**Comment by raymannnnnnnnnnn** [comment]
Hi. All pzem004Tv30 librarys are not compatible with this board. Can you give me a sketch example how to read the pzem004tv30 from the tx/rx uart port? Thanks

**Comment by MatthewJeffson** [comment]
Welcome to our product documentation platform! We value user feedback and contributions, so we couldn’t be more grateful that you can help us improving the documentation.

Hence, if you find obvious errors in the product documentation, or you have additional ideas and insights about using products, please [contribute](https://wiki.seeedstudio.com/Contribution-Guide) to us.

If you need technical support, you can email it to [fae@seeed.cc](mailto:fae@seeed.cc).

If you want to discuss the product, I recommend the [Seeed Studio Forum](https://forum.seeedstudio.com/) and [Seeed Studio Discord Channel](https://discord.gg/eWkprNDMU7)

The "Discussion channel" in Github here is a relatively new addition and may not receive much attention.

Nonetheless, we hope you find it helpful with the help of others. We will clean it up on a monthly basis.

**Comment by dmsherazi** [comment]
i Have a quetion . Can we connect the PCM lines to an ESP device fro direct diagtal audio between the ESp32 and the EC21 module? OR does it require the specific codec chip ?

**Reply by nfs0619** [reply]
Thank you for your question! To better assist you with this technical inquiry, I recommend reaching out to our support team at **[techsupport@seeed.io](mailto:techsupport@seeed.io)**.
